controller/codebase_blackbox_test.go

```diff
@@ -60,44 +63,45 @@ func (s *TestCodebaseREST) SecuredControllers(identity account.Identity) (*goa.S
 }
 
 func (s *TestCodebaseREST) TestSuccessShowCodebaseWithoutAuth() {
-	t := s.T()
-	resource.Require(t, resource.Database)
-
-	cb := requireSpaceAndCodebase(t, s.db)
-
-	svc, ctrl := s.UnsecuredController()
-	_, cbresp := test.ShowCodebaseOK(t, svc.Context, svc, ctrl, cb.ID)
-
-	assert.NotNil(t, cbresp)
-	assert.Equal(t, cb.ID, *cbresp.Data.ID)
-	assert.Equal(t, cb.Type, *cbresp.Data.Attributes.Type)
-	assert.Equal(t, cb.URL, *cbresp.Data.Attributes.URL)
-	assert.Equal(t, cb.LastUsedWorkspace, *cbresp.Data.Attributes.LastUsedWorkspace)
-
+	// Disable gorm's automatic setting of "created_at" and "updated_at"
+	s.DB.Callback().Create().Remove("gorm:update_time_stamp")
```

Comment:
Presumably on the next `Clone` of the DB object. That happens quite often inside of goa.